blaze-bs4/toast/toast.html

```diff
@@ -0,0 +1,46 @@
+<template name='toast'>
+  <style type="text/css">
+  .toasts {
+    position: absolute;
+    right: 30px;
+    z-index:1000;
+    top: 70px;
+  }
+  .hasScrolled {
+    position: fixed;
+    top: 10px;
+  }
+  </style>
+  <div
+    class="toasts" 
+    aria-live="polite" 
+    aria-atomic="true"
+  >
+    <div id="toasts" style="position: absolute; top: 0; right: 0;xz-index:1000;"> </div>
+  </div>
+</template>
```

Comment:
As far as I can see, this is fixing toasts to the upper right corner. Is that correct? We need to support all possible positions. I currently see a good way to use the `add` function. It should also receive some positional attributes like `top`, `left`, `right` and `bottom` and if none are given, it should default to top right.

Please also let's not use `px` but `rem` as unit. Please read here why:

https://stackoverflow.com/questions/37051697/why-bootstrap-4-choose-rem-and-em-instead-px
https://github.com/twbs/bootstrap/issues/19943

Reply:
I changed the px to rem, but frankly can't see how to implement the positional attribute, since it seems tied to the parent div#toasts. Maybe instead of providing the `{{>toasts}}` we change it so people have to put in their own divs and the BlazeBs4.toast.add function needs to be provided the parentNode?

Reply:
I modified the implementation slightly so the `BlazeBs4.toast.add` function now also takes and argument `position`. The default is `top-right` and it also accepts `top-left`,`bottom-right` and `bottom-left`

Reply:
I modified the toast example to show how to use the position

Reply:
looks good to me